Web/Container/.prettierignore

```diff
@@ -0,0 +1 @@
+...ARWidgetsTestProject-main//
```

Comment:
What is this spread syntax? Why do you have doubles slashes at the end of line?

Reply:
Yeah, I think this got changed during a text search and replace. I've removed the file because it wasn't used anyway.